### DISCUSSION TITLE
JsonConfig Error: display.keyWidth has been renamed to display.key.width

### DISCUSSION THREAD

**Discussion by piojpub** [fastfetch-cli/fastfetch#1139] [Q&A] [answered]
Hi,

After updating fastfetch to 2.20.0-1 on Arch Linux, I'm encountering the following error:
 `JsonConfig Error: display.keyWidth has been renamed to display.key.width`

`fastfetch --config ~/.config/fastfetch/config-minimal.jsonc`

[config-minimal.jsonc](https://github.com/user-attachments/files/16447800/config-minimal.jsonc)

Any help resolving this issue would be greatly appreciated.

Thanks a lot!

**Comment by CarterLi** [comment, marked as answer]
Please see the release log

**Reply by piojpub** [reply]
My mistake! 
Thanks for pointing it out.
I'll be more careful next time.
